pkg/services/featuremgmt/toggles_gen.json

```diff
@@ -2204,6 +2204,21 @@
         "requiresRestart": true
       }
     },
+    {
+      "metadata": {
+        "name": "inviteUserExperimental",
```

Comment:
The name and description are a bit generic, are we expecting to reuse this FF to display the button in different places? If not, we should be more specific.